### DISCUSSION TITLE
Why not release new version?

### DISCUSSION THREAD

**Discussion by lyj0309** [distribution/distribution#3757] [Q&A] [answered]
Last release is 7 month ago. Why not release new version?

**Comment by Jamstah** [comment, marked as answer]
We have this backlog at the moment: https://github.com/distribution/distribution/issues?q=is%3Aopen+is%3Aissue+milestone%3ARegistry%2F3.0.0

There may also be more issues, it would probably be sensible to have a community meeting to decide what the requirements for making a next release are.
